### ISSUE TITLE
Can’t connect to Github”

### ISSUE THREAD

**Issue by jd2476** [GitHawkApp/GitHawk#1965]
Hello,

I was able to successfully sign in (with 2FA), and in the settings page I can see I’m logged in, however my inbox isn’t loading with error “cannot connect to GitHub” as a small red notification which pops up from the bottom of the screen, and if I “view all” notifications i get the same error.

Thanks,

Johnny

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.21.0 (1528145114)
Device: iPhone 8 Plus (iOS 11.4.1)
TestFlight: false
</pre>
</details>

**Comment by BasThomas** [comment]
Just to double check, do you have any open notifications on GitHub?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
If you go into Settings and tap “Review GitHub Access”, what happens?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jd2476** [comment]
Hello,

I've been able to solve the problem by logging out and logging in again.

Retracing my steps, I had the following issue:

When I was trying to log into GitHub through GitHawk the first setup, it wasn’t accepting my password (it seemed as if the password was too long).

I activated my account by using the personal access token.

Now, I went back into GitHawk, removed the personal access token, signed out, and signed in again, and it worked.

Thanks,
Johnny
